pkg/web/ui/components/quote_display.templ

```diff
@@ -21,7 +21,7 @@ templ quoteContent(props QuoteDisplayProps) {
 		<div class="size-8 shrink-0">
 			@QuotationMark()
 		</div>
-		<div class="max-h-full overflow-y-auto whitespace-pre-wrap text-left">
```

Comment:
Applying `whitespace-pre-wrap` to the entire div caused it to add unnecessary whitespace at the end of the paragraph. I've moved this to apply it only to the paragraph (which is created by the markdown generator)